ansible/hosts.example.ini

```diff
@@ -151,11 +151,4 @@ bootstrap_os = ubuntu
 # 'flannel' is preferred on bare-metal setups, in case you wish to use `metallb`
 kube_network_plugin = flannel
 
-## You can update server-side versions for helm/kubernetes here on new releases.
-## These server-side versions should match the client-side versions:
-## See download_kubespray.yml for default server-side versions
-## See download_cli_binaries.yml for default client-side versions
-# kube_version = "v1.14.2"
-# helm_version = "v2.13.1"
-## download the kubeconfig after installing to localhost
```

Comment:
suggestion: that line can probably stay

Reply:
Thanks. That last line was removed by accident